npm/react/.releaserc.js

```diff
@@ -1,7 +1,12 @@
+/**
+ * NOTICE
+ * Disable the release for React until further notice.
+ * You may re-enable it by toggling publishConfig: restricted/public
+ * and uncommenting the following lines
+ */
 module.exports = {
-  ...require('../../.releaserc.base'),
-  branches: [
-    { name: '' }, // avoid releasing lastest channel until further notice
-    { name: 'master', channel: 'next', prerelease: 'alpha' },
-  ],
+  // ...require('../../.releaserc.base'),
+  // branches: [
+  //   { name: 'master', channel: 'next', prerelease: 'alpha' },
+  // ],
```

Comment:
I uncommented this and got:

```
A minimum of 1 and a maximum of 3 release branches are required in the branches configuration.

This may occur if your repository does not have a release branch, such as master.

Your configuration for the problematic branches is [].

AggregateError:
    SemanticReleaseError: The release branches are invalid in the `branches` configuration.
```

That seems to be the desired result.